### DISCUSSION TITLE
RemafoX name meaning

### DISCUSSION THREAD

**Discussion by lukemmtt** [FlineDev/RemafoX#19] [General]
@Jeehut 
I'm very curious to know about the origin of the name. I'm a Harry Potter fan too and I've gathered that Bartycrouch is a ref to Barty Crouch Sr's knowledge of a lot of languages, but this one escapes me. Maybe Remus something? Or not Harry Potter related at all?

P.S. I know you pronounce it "reema-fox" in your YouTube videos, but I don't know if I'll ever be able to stop my brain from saying "re-maffo-ex" from the way it's stylized.

**Comment by Jeehut** [comment]
I knew this was incoming. And great to see that you expect it to have something to do with Harry Potter. 😄 

I seriously considered doing that, but then I thought I should use a more professional name for serious apps I make money with, also to make sure I can't get sued for using a name from a big and trademarked franchise like Harry Potter.

This time around, I wanted to have some mystery around the name. The only thing I want to say for now is that there's a reason why some characters are written in uppercase letters and some are not. You can try to guess what it means, if you like, I'm not going to confirm or deny if it's true or not. But be assured that I have some features planned that make the name more obvious, and once they are available, I will reveal the meaning with an animation within the app itself. Something to look out for! 😉

**Comment by Jeehut** [comment]
@lukemmtt FYI: I just changed the name slightly from "ReMafoX" (which you read as "re-maffo-ex") to "RemafoX" which hopefully fixes the issue of how it's been read by some, including you. It doesn't interfere with what it stands for, which I have yet to reveal. :)

**Comment by Jeehut** [comment]
With the introduction of String Catalogs and also [Symbolicated Resources like Images & Colors](https://sarunw.com/posts/swift-symbols-for-asset-catalog/), there's no longer a reason to hide what "RemafoX" stands for, so here's the official reveal, the name stands for "**Re**source **ma**nager **fo**r **X**code".

Actually I had planned to reveal the name with an animation on app start once I had the features implemented to provide safe references to colors and images within Asset Catalogs. But that's no longer needed, so there's no reason to hide the meaning.

I was pretty sad for a few weeks as Apple has not only sherlocked these features I had planned, but String Catalogs also implemented other features RemafoX had shipped with, such as the Linter and the Normalizer. See my [FAQ article](https://www.fline.dev/the-missing-string-catalogs-faq-for-xcode-15/) for more details. But this all frees me up from dealing with low-level stuff and I will focus on more interesting higher-level features in the future!

But the name, well, it doesn't make as much sense as I initially planned. Sorry. 🤷
